microsoft/botframework-solutions
# [Virtual Assistant Template][TypeScript] Create Virtual Assistant Generator

## Description
- Add the base structure of the `generator-botbuilder-assistant`
- Implement `eslint`
- Implement `mocha`

Close #1055 

## Testing Steps

1. Go to `AI\templates\Virtual-Assistant-Template\src\typescript\generator-botbuilder-assistant`
2. Run `npm link` to symlink the package folder
3. Open a terminal wherever you want to create an assistant
4. Run `yo botbuilder-assistant`
5. Follow the generator's prompts
6. Check that the generator finished successfully

## Checklist
\-
